javascript/javascript1/week4/main.js

```diff
@@ -0,0 +1,97 @@
+// Voice Assistant
+
+
+let personName = "";
+ const todoList = [];
+
+ function getReply(command) {
+    if (command === "" || typeof command !== "string") {
+        alert("Give some command Please!");
+    }
+    else if (command.includes("Hello my name is") && personName == "") {
+        personName = command.split(" ").pop();
+        return `Nice to meet you ${personName}`;
+
+    } else if (command.includes("Hello my name is") && personName != "") {
+        return `${personName} Hi again!`;
+
+    } else if (command === "what is my name?") {
+        return `Your name is ${personName}`;
+
+     } if (command === "Add fishing to my todo") {
+         todoList.push("fishing");
+         return todoList[0] + "added to your todoList.";
+
+     } else if (command === "Add singing in the shower to my todo") {
+         todoList.push("Singing");
+         return todoList[1] + "added to your todoList.";
+
+     } else if (command === "What is on my todo?") {
+         return "You have " + todoList.length + " in list to do ' " + todoList + " '";
```

Comment:
What should happen if the `todoList` is `[]`? This will not work with an empty array.